### ISSUE TITLE
clang-format does not understand all regexps

### ISSUE THREAD

**Issue by yjbanov** [angular/angular#4596]
It demands that this:

```typescript
var _urlWithSchemaRe = /^[^:/?#.]+:/g;
```

is formatted as this:

```typescript
var _urlWithSchemaRe = /^[^:/? #.] +: / g;
//                            ^   ^  ^ ^
```

**Comment by mprobst** [comment]
I'll look into it. You can probably work around this for the time being by
escaping the slash, or by instantiating a RegExp object from string.

Yegor <notifications@github.com> schrieb am Mi., 7. Okt. 2015, 14:49:

**Comment by mprobst** [comment]
Have a fix in review: http://reviews.llvm.org/D13641.

**Comment by mprobst** [comment]
Ugh, my fix broke some other behaviour in regexps. Will take a wee bit longer.

**Comment by mprobst** [comment]
OK, this should be fixed by c02f2bdab07904c36bde8bdfa89ef3dd405274b3.
